common/app/common/configuration.scala

```diff
@@ -349,6 +349,9 @@ class GuardianConfiguration extends Logging {
 
   object facebook {
     lazy val appId = configuration.getMandatoryStringProperty("guardian.page.fbAppId")
+    object pages {
+      lazy val authorisedIdsForLinkEdits = configuration.getStringPropertiesSplitByComma("facebook.pages.authorisedIdsForLinkEdits")
```

Comment:
Probably not too often, but I can envisage a new facebook page needing to be added once in a while.